KIF Tests/SystemAlertTests.m

```diff
@@ -21,6 +21,9 @@ + (XCTestSuite *)defaultTestSuite
     if ([UIDevice.currentDevice.systemVersion compare:@"8.0" options:NSNumericSearch] < 0) {
         return nil;
     }
+#if defined(__IPHONE_13_0) // Xcode 11
```

Comment:
Can we use @available instead of `#if`s ?

Reply:
done in https://github.com/openium/KIF/commit/4f286cf6f0264c1b93b5b1f860f00ca0bfb43104